cmd/cloudflared/tunnel/subcommands.go

```diff
@@ -397,6 +399,35 @@ func fmtConnections(connections []cfapi.Connection, showRecentlyDisconnected boo
 	return strings.Join(output, ", ")
 }
 
+func buildReadyCommand() *cli.Command {
+	return &cli.Command{
+		Name:               "ready",
+		Action:             cliutil.ConfiguredAction(readyCommand),
+		Usage:              "Call /ready endpoint and return proper exit code",
+		UsageText:          "cloudflared tunnel [tunnel command options] ready [subcommand options]",
+		Description:        "cloudflared tunnel ready will return proper exit code based on the /ready endpoint",
+		Flags:              []cli.Flag{},
+		CustomHelpTemplate: commandHelpTemplate(),
+	}
+}
+
+func readyCommand(c *cli.Context) error {
+	metricsOpts := c.String("metrics")
```

Comment:
The default value for `--metrics` is `localhost:` which doesn't have the port. Can you add a check here to make sure the caller actually provides the metrics address and port?
```
if !c.IsSet("metrics") {
    return fmt.Errorf("--metrics has to be provided")
}
```

Reply:
Done :+1: